src/api/vdom.js

```diff
@@ -321,6 +321,8 @@ export function element(tname, attrs, ...chren) {
       ch();
     } else if (ctype === 'string' || ctype === 'number') {
       newText(ch);
+    } else if (ctype === 'object') {
```

Comment:
Is this safe enough to check if the type it's an Array? what about `Array.isArray`?